lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -190,6 +194,7 @@ if (typeof module !== 'undefined' && module.exports) {
 /**
  * @typedef {{
  *     type: string,
+ *     styleAs: (string|undefined),
```

Comment:
I had originally implemented styleClass support for arbitrary text elements but it wasn't needed to do just the basic table styling, so I punted to do separately, now we're getting rid of it so won't be added back anyhow